SixTest/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ if(BIGNPART OR HUGENPART)
 endif()
 
 if(NOT COLLIMAT)
-	SET(SIXTRACK_TESTS bbe51 bbe52 bbe571ib0 bb bb_ntwin1 crabamp dipedge distance dynk_globalvars elensidealthin4d elensidealthin6d eric exact frs60 frs javier lostevery lost lostnumxv lostnotilt notilt prob1 prob3 s316 thick4 thick6dblocks tilt elensidealthck4d elensidealthck6d fma thick6ddynk ${SIXTRACK_TESTS})
+	SET(SIXTRACK_TESTS bbe51 bbe52 bbe571ib0 bb bb_ntwin1 crabamp dipedge distance dynk_globalvars elensidealthin4d elensidealthin6d eric exact frs60 frs javier lostevery lost lostnumxv lostnotilt notilt prob1 prob3 s316 thick4 thick6dblocks tilt elensidealthck4d elensidealthck6d fma thick6ddynk wire_thck4d wire_thck6d wire_thin4d wire_thin6d wire_thin6d_ibeco0${SIXTRACK_TESTS})
```

Comment:
Is the lack of space here:
```
wire_thin6d_ibeco0${SIXTRACK_TESTS}
```
between the 0 and the $ okay? Currently ${SIXTRACK_TESTS} starts as empty but it might not stay that way forever.